ui/src/contexts/constants-context.tsx

```diff
@@ -58,6 +62,10 @@ export enum RulePossibilityTranslations {
   NOT_CONTAINS = 'Not Contains (Exact list match)',
   CONTAINS_PARTIAL = 'Contains (Partial list match)',
   NOT_CONTAINS_PARTIAL = 'Not Contains (Partial list match)',
+  COUNT_EQUALS = 'Count Equals',
+  COUNT_NOT_EQUALS = 'Count Does Not Equal',
+  COUNT_BIGGER = 'Count Is Bigger Than',
+  COUNT_SMALLER = 'Count Is Smaller Than',
```

Comment:
gave the copy my best shot... not sure if I was expected to match the existing Title Casing 😅